packages/@okta/vuepress-site/docs/reference/token-hook/index.md

```diff
@@ -124,6 +124,10 @@ When you return an error object, it should have the following structure:
 
 Returning an error object causes Okta to return an OAuth 2.0 error to the requester of the token, with the value of `error` set to `server_error`, and the value of `error_description` set to the string that you supplied in the `errorSummary` property of the `error` object that you returned.
 
+> **Note:** If the error object doesn't include the `errorSummary` property defined, the following common default message is returned to the end user: `The callback service returned an error`
+
+The error message is returned within the redirect uri in either the url query parameter or url fragment, depending on which was set during the initial authentication request.
```

Comment:
This line can be removed, it is already correctly mentioned in L125. We also have other modes, such as form_post and okta_post_message, which will not have the message in the url.